src/rules/onfido-idv.js

```diff
@@ -0,0 +1,108 @@
+/* global configuration */
+
+/**
+ * @title Onfido Identity Verification
+ * @overview Redirect to your Onfido IDV Application for Identity Verification during login.
+ * @gallery true
+ * @category marketplace
+ * 
+ * Onfido digitally prove people’s real identities using a photo ID and facial biometrics. So your users can verify themselves anywhere, anytime. 
+ * By leveraging Onfido with Auth0, we enable organizations to know who their digital user are, tied to a real identity, during authentication, onboarding, other high-risk actions.
+ * This rule will redirect to a custom application you build implementing one of the Onfido Input Capture SDKs. More info: https://developers.onfido.com/
+ * 
+ * Use Rules configuration to define:
+ *    - SESSION_TOKEN_SECRET: Long, random string, should match on Onfido app side.
+ *    - ONFIDO_ID_VERIFICATION_URL: URL to receive the redirect
+ *    - ONFIDO_API_TOKEN: Your Onfido API Token
+ *    - ONFIDO_REGION: The supported Onfido region your tenant is operating in
+ *
+ * @param {object} user
+ * @param {object} context
+ * @param {function} callback
+ */
+async function redirectOnfidoRule(user, context, callback) {
+  // using auth0 rule-utilities to make sure our rule is efficient in the pipeline
+  const { Auth0RedirectRuleUtilities } = require('@auth0/rule-utilities@0.1.0');
+  // requiring Onfido's node SDK for making the calls easier to Onfido's service. 
+  const { Onfido, Region } = require('@onfido/api@1.5.1');
+
+  const ruleUtils = new Auth0RedirectRuleUtilities(user, context, configuration);
+
+  // creating a claim namespace for adding the Onfido IDV check results back to the ID Token
+  const claimNamespace = 'https://claims.onfido.com/';
+
+  // creating a new Onfido client, the region here is where your Onfido instance is located. Possible values are EU for Europe, US for United States, and CA for Canada.
+  const onfidoClient = new Onfido({
+    apiToken: configuration.ONFIDO_API_TOKEN,
+    region:
+      configuration.ONFIDO_REGION === 'EU' ? Region.EU : configuration.ONFIDO_REGION === 'US' ? Region.US : configuration.ONFIDO_REGION === 'CA' ? Region.CA : Region.EU,
+  });
+
+  user.app_metadata = user.app_metadata || {};
```

Comment:
```suggestion
  user.app_metadata = user.app_metadata || {};
  user.app_metadata.onfido = user.app_metadata.onfido || {}
```

Lets you safely check this object later in the Rule.